### PR TITLE
Add prompttools + presentation touch ups

### PR DESCRIPTION
**Pull request by AyushExel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,24 +13,17 @@ Example Applications
...
 | [Audio Search](./examples/audio_search/) | <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/audio_search/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> | [![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)](./examples/audio_search/main.py)  |
 | [Multimodal Image + Text Search](./examples/multimodal_search/) | <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/multimodal_search/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> | [![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)](./examples/multimodal_search/main.py)  |
+| [Evaluating Promps with Prompttools](./examples/prompttools-eval-prompts/) | <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/prompttools-eval-prompts/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> |  |
```
Comment (TevinWang) [inline README.md:16]:
```suggestion
| [Evaluating Prompts with Prompttools](./examples/prompttools-eval-prompts/) | <a href="https://colab.research.google.com/github/lancedb/vectordb-recipes/blob/main/examples/prompttools-eval-prompts/main.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> |  |
```

File: examples/prompttools-eval-prompts/main.ipynb [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+    "# Try with openai\n",
+    "# emb_fns = {\"openai-ada-002\": openai_ada2_embed_func, \"minilm_l6\": minilm_l6_embed_func,  \"default\": default_embed_func }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3babfe5a",
+   "metadata": {},
+   "source": [
+    "Next, we create our test inputs. In this case, we would like to create a new ChromaDB collection.\n",
```
Comment (TevinWang) [inline examples/prompttools-eval-prompts/main.ipynb:88]: Is this supposed to be ChromaDB here?

File: examples/prompttools-eval-prompts/main.ipynb [hunk 3]
```diff
@@ -0,0 +1,762 @@
...
+    "\n",
+    "During the experiment, for each embedding function, a new ChromaDB collection will be temporarily created. The documents will be added into it. Then, we will query from it and examine the results."
```
Comment (TevinWang) [inline examples/prompttools-eval-prompts/main.ipynb:90]: Here as well

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # VectorDB-recipes
-Example Applications
+Examples
```
Comment (TevinWang) [inline README.md:3]: Would be good to add a short description about how these examples are for LanceDB, to get started visit docs, etc.

**Review by TevinWang — APPROVED** [review]
LGTM! Some small things to take a closer look at.

File: examples/prompttools-eval-prompts/main.ipynb (hunk 2 above)
Comment (AyushExel) [reply examples/prompttools-eval-prompts/main.ipynb:88]: Yeah ofc :)

File: README.md (hunk 4 above)
Comment (AyushExel) [reply README.md:3]: Yeah good point.

Comment (AyushExel) [reply README.md:3]: actually, let's not add lancedb at the top as it'll seem marketing-y..We can add it at the bottom maybe?

**Comment by AyushExel** [comment]
@TevinWang updated the things except the links to lancedb. Let me know if this is good to merge

File: examples/prompttools-eval-prompts/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+[Prompttools](https://github.com/hegelai/prompttools) offers a set of free, open-source tools for testing and experimenting with prompts. The core idea is to enable developers to evaluate prompts using familiar interfaces like code and notebooks.
```
Comment (NivekT) [inline examples/prompttools-eval-prompts/README.md:2]:
```suggestion
[Prompttools](https://github.com/hegelai/prompttools) offers a set of free, open-source tools for testing and experimenting with models, prompts, and configurations. The core idea is to enable developers to evaluate prompts using familiar interfaces like code and notebooks. You can use it to experiment with different configurations of LanceDB, and test how LanceDB integrates with the LLM of your choice.
```

**Review by NivekT — COMMENTED** [review]
Thanks for adding `prompttools`!

File: examples/prompttools-eval-prompts/main.ipynb (hunk 2 above)
Comment (AyushExel) [reply examples/prompttools-eval-prompts/main.ipynb]: fixed

**Pull request merged**